components/gallery/GalleryItem.vue

```diff
@@ -345,6 +353,68 @@ function toggleFullscreen() {
   })
 }
 
+function handleReloadClick() {
+  isLoading.value = true
+
+  const reloadElement = (selector: string, isImage?: boolean) => {
+    setTimeout(() => {
+      isLoading.value = false
+      const element: ReloadElement = document.querySelector(selector)
+      if (element) {
```

Comment:
can we put it as one liner here?

```suggestion
      if (element && isImage) {
```

Reply:
I think this causes a typescript error in the `else` statement (`element is null`)